src/components/input/input.js

```diff
@@ -685,6 +687,7 @@ function placeholderDirective($log) {
 
       inputContainer.element.addClass('md-icon-float');
       inputContainer.element.prepend(placeholder);
+      inputContainer.element.toggleClass('md-input-has-label', true);
```

Comment:
This is the same as `inputContainer.element.addClass('md-input-has-label');`. You can also combine it with the `addClass` above into something like:

```js
inputContainer.element
    .addClass('md-icon-float md-input-has-label')
    .prepend(placeholder);
```